### DISCUSSION TITLE
Unwrap in radar or geo coordinates? Does it matter?

### DISCUSSION THREAD

**Discussion by krkrueger** [gmtsar/gmtsar#999] [Q&A]
Basically what the title says, the workflow I have always used with GMTSAR wraps before geocoding, can you use snaphu to unwrap an interferogram that has already been converted to lat/long?

**Comment by Xiaohua-Eric-Xu** [comment]
You could , though we usually recommend doing unwrapping in radar coordinates as 1) better resolution; 2) not affected by re-mosaicing the data; ...
